gustaf/io/nutils.py

```diff
@@ -0,0 +1,147 @@
+"""gustaf/gustaf/io/nutils.py
+
+io functions for nutils.
+"""
+import numpy as np
+
+from gustaf.faces import Faces
+from gustaf.volumes import Volumes
+from gustaf.io.ioutils import abs_fname, check_and_makedirs
+from gustaf.utils import log
+from gustaf.io import mixd
+
+
+def load(fname):
+    """nutils load.
+    Loads a nutils (np.savez) file and returns a Gustaf Mesh.
+
+    Parameters
+    -----------
+    fname: str
+      The npz file needs the following keys:
+      nodes, cnodes, coords, tags, btags, ptags.
+    """
+    npzfile = np.load(fname, allow_pickle=True)
+    nodes = npzfile['nodes']
+    _ = npzfile['cnodes']
+    coords = npzfile['coords']
+    _ = npzfile['tags'].item()
+    btags = npzfile['btags'].item()
+    _ = npzfile['ptags'].item()
```

Comment:
```suggestion
    coords = npzfile['coords']
    btags = npzfile['btags'].item()
```
Are these unused variables for future-proofing or can they be ignored?

Reply:
These are all the parameters, which for the ```nutils.mesh.simplex``` function https://docs.nutils.org/en/v7.0/nutils/mesh/?highlight=simplex#nutils.mesh.simplex. So yes, they are more like unused variables for future - and we want to ignore the empty variables so far.